src/scripts.js

```diff
@@ -135,7 +139,7 @@ filter.addEventListener('input', event => {
   }
 })
 
-filterClearButton.addEventListener('click', () => {
+function clearFilterByTag() {
```

Comment:
This is clever! Very clean way to handle the UX issue. Later on, since it's currently sitting in the Event Listeners section, we might want to move `clearFilterByTag()` into a new Event Handler section or into the Utility Functions section.

Reply:
Good call! I'll make myself a ticket to move that function to the correct place.